javascript-forms/form-values.js

```diff
@@ -0,0 +1,10 @@
+const $form = document.forms[0];
+$form.addEventListener('submit', function (event) {
+  event.preventDefault();
+  const messageData = {};
+  for (let i = 0; i < $form.elements.length - 1; i++) {
+    messageData[$form.elements[i].name] = $form.elements[i].value;
+  }
```

Comment:
While this may work here, it is finnicky, since there can be multiple elements with the same `name` in the same form. Get each input by its `name` property:

---

![image](https://user-images.githubusercontent.com/16143771/138156571-c7df98a2-d648-4484-89ae-538d3e2d4199.png)